django/conf/global_settings.py

```diff
@@ -502,6 +502,17 @@
 CACHE_MIDDLEWARE_SECONDS = 600
 CACHE_MIDDLEWARE_ALIAS = 'default'
 
+#########
+# TASKS #
+#########
+
+# The task backends to use.
+QUEUES = {
```

Comment:
But in `DATABASES` you define databases and in `CACHES` you define caches, but here you do not define tasks but entities that take tasks. On the other hand, this would be like the `TEMPLATES` setting...